### PR TITLE
[Heartbeat] Move changelog entries from breaking to bug

### PR DESCRIPTION
**Pull request by andrewvc**
These were in the wrong place by mistake

### CODE REVIEW

**Review by vigneshshanmugam — APPROVED** [review]

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -122,6 +117,11 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 
 - Fix race condition in http monitors using `mode:all` that can cause crashes. {pull}29697[pull]
 - Fix broken ICMP availability check that prevented heartbeat from starting in rare cases. {pull}29413[pull]
+- Fix broken macOS ICMP python e2e test. {pull}29900[29900]
+- Only add monitor.status to browser events when summary. {pull}29460[29460]
+- Also add summary to journeys for which the synthetics runner crashes. {pull}29606[29606]
+- Update size of ICMP packets to adhere to standard min size. {pull}29948[29948]
+- Add fonts to support more different types of characters for multiple languages. {pull}29606[29861]
```
Comment (lucasfcosta) [inline CHANGELOG.next.asciidoc:124]:
```suggestion
- Add fonts to support more different types of characters for multiple languages. {pull}29861[29861]
```

Seems like this was the right link but wrong named reference:

See: https://github.com/elastic/beats/pull/29861

**Review by lucasfcosta — COMMENTED** [review]
LGTM, but I did catch one misreference I should've seen before.

**Comment by andresrc** [comment]
I fixed it directly in https://github.com/elastic/beats/pull/30111, I think we can close this

**Comment by andrewvc** [comment]
@andresrc shouldn't we still correct the issue in master, or do you delete these entries once 7.17.0 is released

**Comment by andresrc** [comment]
@andrewvc I take care of moving things to master, we can close this one, thanks!

**Comment by andresrc** [comment]
Closing this as it has been fixed as part of the 7.17 (including forward ports) and 8.0 PRs

**Pull request closed without merging**